stubs/yt-dlp/METADATA.toml

```diff
@@ -0,0 +1,3 @@
+version = "2025.05.*"
```

Comment:
Considering this uses "real" calver, I would pin this to the exact version. There is no semantic meaning to the version, after all.